### DISCUSSION TITLE
Subaru Solterra Support

### DISCUSSION THREAD

**Discussion by seang96** [G-Two/homeassistant-subaru#106] [Q&A]
It looks like Subaru solterra uses its own service / app, probably because it's software was made by Toyota. Is there any possibility of adding support for this?

**Comment by stboch** [comment]
Have you tried to see if this works? https://github.com/widewing/ha-toyota-na

Looking over the app it looks like while they also use Sirius XM telematics this app won't be able to be made compatible might need to modify the Toyota app to work.
